mlir/lib/Dialect/SPIRV/IR/SPIRVOps.cpp

```diff
@@ -875,7 +882,9 @@ ParseResult spirv::ExecutionModeOp::parse(OpAsmParser &parser,
     }
     values.push_back(llvm::cast<IntegerAttr>(value).getInt());
   }
-  result.addAttribute(kValuesAttrName,
+  StringRef valuesAttrName =
+      spirv::ExecutionModeOp::getValuesAttrName(result.name).strref();
```

Comment:
StringRef still?

Reply:
whoops.
Otherwise good though?

Reply:
I only superficially looked, this will be more for @antiagainst to approve I think.

I would mentioned that this PR would benefit from a description (the title says the "what" is done, the description should say the "why" and provide context).
(See https://mlir.llvm.org/getting_started/Contributing/#commit-messages )